### DISCUSSION TITLE
What does LazyVim use as vim.ui backend for code action menu

### DISCUSSION THREAD

**Discussion by zayihu** [LazyVim/LazyVim#4352] [General]
I don't really understand if LazyVim uses dressing.nvim for this or noice.nvim or something different.

**Comment by RayGuo-ergou** [comment]
fzf-lua

See https://github.com/LazyVim/LazyVim/commit/5c876b754999eb5fe8b79c5b6f5249b5bbddb454

**Reply by dpetka2001** [reply]
This only happens when fzf Extra is enabled. Otherwise `dressing.nvim` is used.

**Reply by RayGuo-ergou** [reply]
Yes, indeed. Sorry I somehow thought `fzf-lua` is the default picker.
